lib/pages/chat/chat_view.dart

```diff
@@ -115,15 +116,8 @@ class ChatView extends StatelessWidget {
     controller.sendingClient ??= client;
     controller.room = controller.sendingClient!.getRoomById(controller.roomId!);
     if (controller.room == null) {
-      return Scaffold(
-        appBar: AppBar(
-          toolbarHeight: 120,
-          title: Text(L10n.of(context)!.oopsSomethingWentWrong),
-        ),
-        body: Center(
-          child: Text(L10n.of(context)!.youAreNoLongerParticipatingInThisChat),
-        ),
-      );
+      context.go('/error');
+      return const LinearProgressIndicator();
```

Comment:
Why don't you `return const SizedBox()`

Reply:
I could, but thought a progress was more suitable